docs/book/architecture.rst

```diff
@@ -44,11 +66,13 @@ API is exactly the same and you are probably already familiar with it:
     {
         $manager = $this->container->get('sylius.manager.product');
 
-        $manager->persist($product1);
-        $manager->remove($product2);
-        $manager->flush(); // Save changes in database.
+        // Assuming that there exists the $product1 in the database we can perform such operations:
+        $manager->remove($product1);
+        $manager->persist($product2);
+        $manager->flush(); // Save changes in the database.
```

Comment:
Somehow i've missed the fact that it's a manager :D
Still explanation what happened would be nice